scene/3d/lightmap_gi.cpp

```diff
@@ -992,7 +999,14 @@ LightmapGI::BakeError LightmapGI::bake(Node *p_from_node, String p_image_data_pa
 		config->set_value("remap", "importer", "2d_array_texture");
 		config->set_value("remap", "type", "StreamTexture2DArray");
 		if (!config->has_section_key("params", "compress/mode")) {
-			config->set_value("params", "compress/mode", 2); //user may want another compression, so leave it be
+			// The user may want another compression method, so leave it be if it was customized.
+			if (use_hdr) {
+				config->set_value("params", "compress/mode", 2); // COMPRESS_VRAM_COMPRESSED
+			} else {
+				// Low dynamic range lightmaps look bad when using VRAM compression,
+				// so use lossless compression by default instead.
+				config->set_value("params", "compress/mode", 0); // COMPRESS_LOSSLESS
+			}
```

Comment:
It seems VRAM compression has no effect for EXR files in `master` (unlike `3.x`). Should always I use `COMPRESS_LOSSLESS` as is done in the `3.x` branch? VRAM compression still looks quite bad in `3.x` on EXR lightmaps, so it may be better to default to lossless compression. 

We could also default to lossy WebP compression (for LDR lightmaps at least), but now that we have lossless WebP, the gap between lossless and lossy compression is smaller.

Reply:
The reason VRAM compression appears to have no effect is because only BPTC allows VRAM compression and it is disabled by default.